Firebase/Core/Private/FIRHeartbeatInfo.h

```diff
@@ -0,0 +1,27 @@
+//
+//  HeartbeatInfo.h
+//  AppHost-FirebaseInstanceID-iOS-Unit-Tests
+//
+//  Created by Vinay Guthal on 10/17/19.
+//
+
+#import <Foundation/Foundation.h>
+
+NS_ASSUME_NONNULL_BEGIN
+
+@interface FIRHeartbeatInfo : NSObject
+
+typedef NS_ENUM(NSInteger, Heartbeat) {
```

Comment:
Here is a [good example](https://github.com/firebase/firebase-ios-sdk/blob/fb441a0376b0343d35ff36c18ee246f40f5be2b3/Firebase/Core/Public/FIRLoggerLevel.h#L23) of defining an enum. So the enum name should be like `FIRHeartbeatInfoCode` with the cases named like `FIRHeartbeatInfoCodeNone`, etc. 
`NS_SWIFT_NAME` is not necessary (not a public API) but encouraged in this case.